src/Makefile.mk

```diff
@@ -54,7 +54,7 @@ CFLAGS_LINT ?=
 LINKER ?= $(CC)
 LDFLAGS ?=
 LDOUTF ?= -o
-LIBS ?=
+LIBS := -lusb-1.0
```

Comment:
It looks like this is not conditional on KLH10_DEV_LITES. Can you make a configure option for the lights, which controls the addition of libusb and -DKLH10_DEV_LITES? It could check for libusb and if not present, refuse the lights option. Or if you prefer, I can give it a go after merging this.

Reply:
If you offer to, yes please add the configure option.

Reply:
Ok, I'll do that ~soon after merging then.